src/event-map.js

```diff
@@ -0,0 +1,350 @@
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    encrypted: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    ended: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    loadedData: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    loadedMetadata: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    loadStart: {
+      EventType: 'ProgressEvent',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    pause: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    play: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    playing: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    progress: {
+      EventType: 'ProgressEvent',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    rateChange: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    seeked: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    seeking: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    stalled: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    suspend: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    timeUpdate: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    volumeChange: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    waiting: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    // Image Events
+    load: {
+      EventType: 'UIEvent',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    error: {
+      EventType: 'Event',
+      defaultInit: {bubbles: false, cancelable: false},
+    },
+    // Animation Events
+    animationStart: {
+      EventType: 'AnimationEvent',
+      defaultInit: {bubbles: true, cancelable: false},
+    },
+    animationEnd: {
+      EventType: 'AnimationEvent',
+      defaultInit: {bubbles: true, cancelable: false},
+    },
+    animationIteration: {
+      EventType: 'AnimationEvent',
+      defaultInit: {bubbles: true, cancelable: false},
+    },
+    // Transition Events
+    transitionEnd: {
+      EventType: 'TransitionEvent',
+      defaultInit: {bubbles: true, cancelable: true},
+    },
+    // pointer events
+    pointerOver: {
+      EventType: 'PointerEvent',
+      defaultInit: {bubbles: true, cancelable: true, composed: true},
+    },
+    pointerEnter: {
+      EventType: 'PointerEvent',
+      defaultInit: {bubbles: false, cancelable: false},
```

Comment:
I'm deferring to https://w3c.github.io/pointerevents/index.html#attributes-and-default-actions, but I noticed that Chromium-based browsers are non-conformant and set these as composed, while at least Firefox seems conformant. Calling it out in case you have a different preference.